src/main/java/org/openlmis/requisition/service/RequisitionService.java

```diff
@@ -508,7 +508,7 @@ public void checkIfCanApproveRequisition(UUID programId, UUID supervisoryNodeId,
   public Page<RequisitionWithSupplyingDepotsDto>
       searchApprovedRequisitionsWithSortAndFilterAndPaging(String filterValue,
                                                            String filterBy,
-                                                           String sortBy,
+                                                           List<String> sortBy,
```

Comment:
Then I need to add sort field to **paginated** trait type (api-definition). The problem is that this type is used in 4 endpoints. I am not sure if all of them handle sorting.